prime-angular-frontend/src/app/modules/enrolment/shared/components/job-form/job-form.component.ts

```diff
@@ -12,6 +12,9 @@ import { Job } from '@enrolment/shared/models/job.model';
   templateUrl: './job-form.component.html',
   styleUrls: ['./job-form.component.scss']
 })
+/**
+ * @deprecated Due to moving Job Title to Obo Site (PRIME-1459)
```

Comment:
If this is deprecated and unused we can drop the code and component

Reply:
@neophyte57 I've made a note of this so we can deal with it next sprint with other job related clean up tasks.